features/org.eclipse.e4.tools.persistence.feature/feature.xml

```diff
@@ -2,7 +2,7 @@
 <feature
       id="org.eclipse.e4.tools.persistence.feature"
       label="%featureName"
-      version="1.1.300.qualifier"
+      version="1.1.301.qualifier"
```

Comment:
Is this version-bump necessary?

Reply:
as per https://ci.eclipse.org/pde/job/eclipse.pde/job/PR-1459/4/console - though it is a warning